library-model-loader/build.gradle

```diff
@@ -35,3 +35,15 @@ dependencies {
     annotationProcessor deps.build.nullaway
     compileOnly deps.build.guava
 }
+
+import net.ltgt.gradle.errorprone.CheckSeverity
+
+tasks.withType(JavaCompile) {
+    // remove the if condition if you want to run NullAway on test code
+    if (!name.toLowerCase().contains("test")) {
+        options.errorprone {
+            check("NullAway", CheckSeverity.ERROR)
+            option("NullAway:AnnotatedPackages", "edu")
+        }
+    }
+}
```

Comment:
Please end file in new line

Reply:
5802e7e7ec2c5d251813668e0a58f86204ddd1d9